en/docs/how-tos/service-discovery.md

```diff
@@ -43,190 +41,16 @@ Enable service discovery using etcd on WSO2 API Microgateway as follows:
-
-        <table>
-        <thead>
-        <tr class="header">
-        <th><br />
-        </th>
-        <th>Base64 Decoded Value</th>
-        <th>Base64 Encoded Value</th>
-        </tr>
-        </thead>
-        <tbody>
-        <tr class="odd">
-        <td>Key</td>
-        <td>petstore</td>
-        <td><code>                      cGV0c3RvcmU=                     </code></td>
-        </tr>
-        <tr class="even">
-        <td>Value</td>
-        <td><a href="http://petstore.swagger.io/v2/" class="uri">http://petstore.swagger.io/v2/</a></td>
-        <td><code>                      aHR0cDovL3BldHN0b3JlLnN3YWdnZXIuaW8vdjIv                     </code></td>
-        </tr>
-        </tbody>
-        </table>
-
-
-        ``` java tab="Example"
-        curl -L http://localhost:2379/v3/kv/put -X POST -d '{"key": "cGV0c3RvcmU=", "value": "aHR0cDovL3BldHN0b3JlLnN3YWdnZXIuaW8vdjIv"}' -H 'Authorization : UOkLazCNAoHnxiDz.11'
-        ```
         
-        ``` java tab="Sample Response"
-        {"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"2","raft_term":"2"}}
-        ```
-        
-        !!! info
-            If you want to check whether you have stored your key-value pair correctly, run the following command.
-
-        **Format**
+    3.  Again navigate to the ```<etcd_HOME>``` and put the enpoint URL to the etcd server using etcdctl tool, by executing the following command.
         
-        ``` java 
-            curl -L <etcd-client-URL>/<etcd-cli-version>/kv/range -X POST -d '{"key": "<base64-encoded-key>"}' -H 'Authorization : <auth-token>'
-        
-            - <etcd-client-URL> - You can find your client URL in the terminal that you used to start the etcd server.
-            - <etcd-cli-version> - The CLI version that corresponds to your etcd version. For example, `/v3alpha` is the CLI version that corresponds to etcd v3.3.13.
-            - <base64-encoded-key> - Encode the key using a base64 encoder (e.g., <https://www.base64encode.org/> ) and enter the respective value.
-            - <base64-encoded-value> - Encode the endpoint URL using a base64 encoder and enter the respective value. 
-            - <auth-token> - Enter the Auth token that you obtained in [step c](#ServiceDiscovery-AuthToken) .
-        ```
-        **Sample**
-        
-        Let's use the following value for this example.
-        <table>
-        <thead>
-        <tr class="header">
-        <th><br />
-        </th>
-        <th>Base64 Decoded Value</th>
-        <th>Base64 Encoded Value</th>
-        </tr>
-        </thead>
-        <tbody>
-        <tr class="odd">
-        <td>Key</td>
-        <td>petstore</td>
-        <td><code>                      cGV0c3RvcmU=                     </code></td>
-        </tr>
-        </tbody>
-        </table>
-            
-            
+        ``` java tab="Format"
+        ./etcdctl put <Key> <Value>        
```

Comment:
Remove `./` from all commands. Expect `etcdctl` to be install properly in the environment
```suggestion
        etcdctl put <Key> <Value>
```